wandb/proto/wandb_internal.proto

```diff
@@ -150,6 +151,11 @@ message GitRepoRecord {
   string commit = 2;
 }
 
+message WandbConfigParametersRecord {
```

Comment:
We have a pattern of underdocumented code, but I'm trying to change that. Could you add comments? Protobuf documentation comments just use double slashes `//`, and follow the usual style of

* A summary line, less than 80 characters total
* Optional details, separated by a blank line from the summary

Comments get included in the generated code and can be displayed by an IDE on hover.

Here are some things that aren't obvious:

* `include` and `exclude` are "paths" to values in a run's config
* What is this record used for?

Reply:
(your PR description is great for this, just turn it into documentation)